frontend/src/metabase/visualizations/types/visualization.ts

```diff
@@ -114,17 +115,15 @@ export type VisualizationGridSize = {
   height: number;
 };
 
-// TODO: add component property for the react component instead of the intersection
-export type Visualization = React.ComponentType<VisualizationProps> & {
-  name: string;
-  noun: string;
+export type VisualizationStaticProps = {
```

Comment:
Similar to with `staticProps`, let's name this something like `VisualizationProperties` instead to avoid confustion with React props.

Reply:
66fbffd1a83e57f4fc3a7a18413c5343e2bb7a1a